### PR TITLE
Tidying up 

### PR DESCRIPTION
**Pull request by RaaviSindhu**
Adjusted text boxes of player 1 and 2 to display above the grid

### CODE REVIEW

File: CodeBenders/ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.10f1
-m_EditorVersionWithRevision: 2019.4.10f1 (5311b3af6f69)
+m_EditorVersion: 2019.4.11f1
+m_EditorVersionWithRevision: 2019.4.11f1 (2d9804dddde7)
```
Comment (bharatht96) [inline CodeBenders/ProjectSettings/ProjectVersion.txt:2]: Should we be using this newer version or stay consistent with the current version?

Comment (late-in-autumn) [reply CodeBenders/ProjectSettings/ProjectVersion.txt:2]: I believe that all versions between 2019.4 are inter-compatible. This is just a patch version and we can all upgrade to the latest iteration for bugfixes (IIRC the latest version now is 2019.4.12f1).

**Review by bharatht96 — APPROVED** [review]

**Comment by late-in-autumn** [comment]
If you set the preview (go to game and set the aspect ratio to either 16:9 or 186:100) other than freeform, you'll see that the text and buttons are clipped in the build mode:

clipped buttons

**Review by late-in-autumn — CHANGES_REQUESTED** [review]
If you set the preview (go to game and set the aspect ratio to either 16:9 or 186:100) other than freeform, you'll see that the text and buttons are clipped in the build mode:

clipped buttons

**Comment by late-in-autumn** [comment]
Seems it is now included in #28 so I am closing this one to avoid confusion.

**Pull request closed without merging**